docs/manual/get_executable.md

```diff
@@ -74,7 +74,10 @@ $ git clone https://github.com/FISCO-BCOS/FISCO-BCOS.git
 $ cd FISCO-BCOS
 $ git checkout master
 $ mkdir -p build && cd build
-# CentOS请使用cmake3
```

Comment:
This should not change, the first section is compiling the source code, the doc section aims to generate doc